src/basic/alloc-util.c

```diff
@@ -102,3 +102,8 @@ void* greedy_realloc0(
 
         return q;
 }
+
+void *expand_to_usable (void *ptr, size_t newsize _unused_)
+{
```

Comment:
as per coding style, we place the opening `{` for functions on the same line as the function name, i.e. remove one newline here.

and no space between function name and `(`, as per coding style.

https://systemd.io/CODING_STYLE/